File: BOJ/11399.js
let fs = require("fs");
let input = fs.readFileSync("/dev/stdin").toString().trim().split("\n");

const N = Number(input[0]);
input = input[1]
  .split(" ")
  .map(Number)
  .sort((a, b) => a - b);

let answer = 0;
let sum = 0;

for (let i = 0; i < N; i++) {
  sum += input[i];
  answer += sum;
}

console.log(answer);

File: baekjoon/11399.py
# N명의 사람들이 ATM에서 돈을 인출
# 한 사람의 인출 시간은 P분
# 사람들이 서있는 순서에 따라 인출 시간이 달라짐
# 총 인출 시간을 최소로

n = int(input())
peoples = list(map(int, input().split()))
peoples.sort()

answer = 0
sum = 0

# sum을 계속 더해가면서 answer에 추가해준다.
for i in range(n):
  sum += peoples[i]
  answer += sum

print(answer)